build/notebooks/table_variants_missense.md

```diff
@@ -5070,7 +5070,246 @@ tbl_variants_selected.displayall(tr_style=tr_style, td_styles=td_styles)
 
 
 ```python
+# tbl_variants_display = (
+#     tbl_variants_selected
+#     .cut(['POS', 'REF', 'ALT', 'FILTER_PASS', 'AGAP004707-RA'] + ['AF_' + p for p in pop_ids])
+#     .convert(['AF_' + p for p in pop_ids], lambda v: int(np.rint(v * 100)))
+#     .addfield('substitution', lambda row: '{:,} {}>{}'.format(row['POS'], row['REF'], row['ALT']), index=0)
+#     .convert('substitution', lambda v, row: v + '*' if not row['FILTER_PASS'] else v, pass_row=True)
+#     .convert(['substitution', 'AGAP004707-RA'], lambda v: r'\texttt{%s}' % v)
+#     .cutout('POS', 'REF', 'ALT', 'FILTER_PASS')
+# )
+# tbl_variants_display.displayall()
+```
+
+
+```python
+# prologue = r"""
+# \begin{tabular}{lllrrrrrrrrrrr}
+# \toprule
+# \multicolumn{3}{c}{Mutation} &
+# \multicolumn{9}{c}{Population allele frequency (\%)} &
+# \multicolumn{2}{c}{LD ($D'$)} \\
+# \cmidrule(r){1-3}
+# \cmidrule(r){4-12}
+# \cmidrule(r){13-14}
+# Position\tablefootnote{Position relative to AgamP3 reference sequence, chromosome arm 2L.} & 
+# \emph{Ag}\tablefootnote{Codon numbering according to transcript \texttt{AGAP004707-RA} in geneset AgamP4.4.} & 
+# \emph{Md}\tablefootnote{Codon numbering according to @@TODO.} & 
+# AO\emph{Ac} & 
+# BF\emph{Ac} & 
+# GN\emph{Ag} & 
+# BF\emph{Ag} & 
+# CM\emph{Ag} & 
+# GA\emph{Ag} & 
+# UG\emph{Ag} & 
+# KE & 
+# GW & 
+# \texttt{L995F} & 
+# \texttt{L995S} \\
+# \midrule
+# """
+# template = r"""
+# {substitution} & {AGAP004707-RA} & NA & {AF_AOM} & {AF_BFM} & {AF_GNS} & {AF_BFS} & {AF_CMS} & {AF_GAS} & {AF_UGS} & {AF_KES} & {AF_GWA} & NA & NA \\
+# """
+# epilogue = r"""
+# \bottomrule
+# \end{tabular}
+# """
+# tbl_variants_display.totext('../tables/variants_missense.tex', 
+#                             encoding='ascii',
+#                             prologue=prologue, 
+#                             template=template,
+#                             epilogue=epilogue)
+```
+
+
+```python
+# !cat ../tables/variants_missense.tex
+```
+
+# add housefly numbering
+
+
+```python
+#to easily split the codon numbers from the codon letters
+import re
+```
+
+
+```python
+#load the codon map from the blog post (with the header info removed)
+md_tbl = etl.fromtsv('/home/chris/Git/agam-vgsc-report/data/domestica_gambiae_map.txt')
+
+#get codons
+dom = list(md_tbl['domestica_codon'])
+ano = list(md_tbl['gambiae_codon'])
+```
+
+
+```python
+#make a dictionary with a as the key and d as the value
+map_dict = {a: d for a, d in zip(ano, dom)}
+```
+
+
+```python
+#get the snpeff annotations
+gam_cod = list(tbl_variants_selected['SNPEFF_HGVS_p'])
```

Comment:
If you use the 'AGAP004707-RA' column rather than the 'SNPEFF_HGVS_p' column then you should have the data for the multiallelics, which means you won't have to do a manual fix later on.